plugins/view-transitions/includes/theme.php

```diff
@@ -42,4 +103,44 @@ function plvt_load_view_transitions(): void {
 	wp_register_style( 'wp-view-transitions', false, array(), null ); // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
 	wp_add_inline_style( 'wp-view-transitions', $stylesheet );
 	wp_enqueue_style( 'wp-view-transitions' );
+
+	$theme_support = get_theme_support( 'view-transitions' );
+
+	/*
+	 * No point in loading the script if no specific view transition names are configured.
+	 */
+	if (
+		( ! is_array( $theme_support['global-transition-names'] ) || count( $theme_support['global-transition-names'] ) === 0 ) &&
+		( ! is_array( $theme_support['post-transition-names'] ) || count( $theme_support['post-transition-names'] ) === 0 )
```

Comment:
No need for the `is_array()` checks if the theme support value has been sanitized, right?

```suggestion
		( count( $theme_support['global-transition-names'] ) === 0 ) &&
		( count( $theme_support['post-transition-names'] ) === 0 )
```

Reply:
Not really, but since the data comes from a global variable, I think it's better to use defensive coding.